armi/bookkeeping/report/reportInterface.py

```diff
@@ -120,11 +120,11 @@ def interactEOL(self):
         b = self.o.r.core.getFirstBlock(Flags.FUEL)
         b.setAreaFractionsReport()
 
-        from armi.bookkeeping import plotting
+        from armi.utils import reportPlotting
```

Comment:
Does this import need to be inside the method due to circular import issues? I can't tell right away.

Reply:
Yeah, that WAS to prevent a circular import. But I can fix it now.